appveyor.yml

```diff
@@ -94,7 +94,7 @@ build_script:
     cd %ROOT_DIRECTORY% &&
     mkdir build &&
     cd build &&
-    cmake -DBOOST_LIBRARYDIR=%BOOST_ROOT%\stage\lib -DCMAKE_CXX_STANDARD=17 -Dbuild_tests=ON -Dbuild_examples=ON -Dbuild_tools=ON -Dpython-bindings=%python% -Dboost-python-module-name="python" -Dskip-python-runtime-test=true -DPython_ADDITIONAL_VERSIONS="2.7" -G "Visual Studio 16 2019" -A x64 .. &&
+    cmake -DBOOST_LIBRARYDIR=%BOOST_ROOT%\stage\lib -DCMAKE_CXX_STANDARD=17 -Dbuild_tests=ON -Dbuild_examples=ON -Dbuild_tools=ON -Dwebtorrent=%webtorrent% -Dpython-bindings=%python% -Dboost-python-module-name="python" -Dskip-python-runtime-test=true -DPython_ADDITIONAL_VERSIONS="2.7" -G "Visual Studio 16 2019" -A x64 .. &&
```

Comment:
I don't think this is enough. `webtorrent` defaults to `off`. I think you have to add it to the build matrix too

Reply:
It's already on in the build matrix:
https://github.com/arvidn/libtorrent/blob/ee7bd3b109d2d63d662486d707c33b5777717ed3/appveyor.yml#L26

Reply:
but that's not for the `cmake` build.
right under:
```
- cmake: 1
```

add:

```
  webtorrent: on
```

Reply:
Oh, yes...